ansible/roles/dhcrelay/handlers/main.yml

```diff
@@ -0,0 +1,9 @@
+---
+# Copyright (C) 2020 CipherMail B.V. <https://www.ciphermail.com/>
+# Copyright (C) 2020 DebOps <https://debops.org/>
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+- name: Restart isc-dhcp-relay
+  service:
+    name: 'isc-dhcp-relay'
+    state: 'restarted'
```

Comment:
The handlers should now be added in the `global_handlers` role, in a separate file. Don't forget to import that role in the task list.